adaptdl_cli/adaptdl_cli/registry.py

```diff
@@ -75,15 +93,14 @@ def _fix_daemon_json():
 
 
 def registry_running():
-    svc = json.loads(subprocess.check_output(['kubectl', 'get', 'svc', '-l',
-                                             'app=docker-registry', '-ojson']))
+    svc = json.loads(subprocess.check_output(['kubectl', 'get', 'svc',
+                                              '-A', '-l',
+                                              'app=docker-registry',
+                                              '-ojson']))
```

Comment:
Possible to do a more targeted/specific query by adding an annotation to the registry adaptdl uses? i.e https://github.com/helm/charts/blob/master/stable/docker-registry/values.yaml#L29

Reply:
yes